reset.css

```diff
@@ -0,0 +1,42 @@
+*{
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
```

Comment:
small suggestions - instead of decalaring the reset.css again. Can we have a main.css file in component library that has all styles and we link it here